src/lib/index.css

```diff
@@ -1,7 +1,6 @@
 html,
 body {
   font-family: 'Lato';
-  font-size: max(14px, 0.972vw);
```

Comment:
Let's keep this line for now (it can have unexpected layout impacts to remove it now)